app/src/main/res/layout/contact_list_entry.xml

```diff
@@ -24,15 +24,9 @@
         <TextView
             android:id="@+id/name"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            android:layout_height="match_parent"
             android:text="@string/name"
-            android:textSize="@dimen/contacts_list_entry_text_name_size" />
-
-        <TextView
-            android:id="@+id/pubKeySnippet"
```

Comment:
I still think we should keep this... Maybe you know the first couple of characters of someone's key, and this gives more confidence you got the right person